src/global.css

```diff
@@ -37,6 +37,10 @@ body {
   gap: 1rem;
 }
 
+button {
```

Comment:
i think this style should be more specific and not be applied to all buttons, it might overwrite other code.